src/components/notifications/static/static.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import Static from './static';
+
+describe('Static', () => {
+  it('should pass data-attributes all way to the DOM', () => {
+    const { container } = render(
+      <Static data-foo="bar" type="info">
+        A message
+      </Static>
+    );
+
+    expect(container.querySelector("[data-foo='bar']")).toBeInTheDocument();
+  });
+
+  it.each(['error', 'info', 'success', 'warning'])(
+    'should render notification of type `%s`',
+    type => {
+      const { container } = render(<Static type={type}>A message</Static>);
+
+      expect(container).toMatchSnapshot();
```

Comment:
This test will break whenever we change the implementation because it asserts the whole DOM. This is pretty inflexible and what we're trying to get away from by using `react-testing-library`.

Can you write this test in another way? What is the actual thing being tested here?